app/assets/stylesheets/application.scss

```diff
@@ -24,11 +24,27 @@
 
 // font-family: 'Quicksand', sans-serif;
 
+body {
+  font-family: "Titillium Web", sans-serif;
+}
+
 h1 {
   font-family: "Lobster";
 }
 
+strong {
+  color:  #c62828;
+  font-weight: 700;
+}
+
+nav {
+  background-color: rgb(100, 100, 100);
```

Comment:
Color literals like `rgb(100, 100, 100)` should only be used in variable declarations; they should be referred to via variable everywhere else.